modules/apps/commerce/commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/helper/ProductHelperImpl.java

```diff
@@ -292,7 +292,12 @@ private boolean _isTaxIncludedInPrice(long commerceChannelId)
 		throws PortalException {
 
 		CommerceChannel commerceChannel =
-			_commerceChannelLocalService.getCommerceChannel(commerceChannelId);
+			_commerceChannelLocalService.fetchCommerceChannel(
```

Comment:
why? a channel should always be there
having no channel doesn't mean tax are not included in the price